src/DocBlock/Tags/InvalidTag.php

```diff
@@ -104,6 +104,7 @@ static function (&$value) : void {
         do {
             $trace = array_map(
                 static function (array $call) use ($flatten) : array {
+                    $call += ['args' => []];
```

Comment:
Instead of this, why not change the line below to `$call['args'] ?? []`?

Reply:
In fact, this might be the only correct fix. I think args did exist, but was null, based on the error you linked to?

Reply:
> Instead of this, why not change the line below to `$call['args'] ?? []`?

Using `$call['args'] ?? []` is also a valid solution, one or the other is fine :-)

> In fact, this might be the only correct fix. I think args did exist, but was null, based on the error you linked to?

It's ok for me then I can update.

Reply:
> Using $call['args'] ?? [] is also a valid solution, one or the other is fine :-)

Only my one is correct, I think. It also fixes the case where args is set but is null, which is the bug you were experiencing?

Reply:
You are perfectly right. Thanks for pointing that out.